File: HostAndGuest_JavProject/src/com/hostandguest/entities/Booking.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.hostandguest.entities;

import java.util.Date;
import java.util.Objects;

/**
 *
 * @author The
 */
public class Booking {
    private int id;
    private int term;
    private boolean reviewed;
    private Date bookingDate;
    private int nbr_rooms_reserved;
    private double total_amount;
    private boolean isGift;
  
    // Many To One 
    private Property property;
    
    // Many To One
    private User guest;


    public Booking() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getTerm() {
        return term;
    }

    public void setTerm(int term) {
        this.term = term;
    }

    public boolean isReviewed() {
        return reviewed;
    }

    public void setReviewed(boolean reviewed) {
        this.reviewed = reviewed;
    }

    public Date getBookingDate() {
        return bookingDate;
    }

    public void setBookingDate(Date bookingDate) {
        this.bookingDate = bookingDate;
    }

    public Property getProperty() {
        return property;
    }

    public void setProperty(Property property) {
        this.property = property;
    }

    public User getGuest() {
        return guest;
    }

    public void setGuest(User guest) {
        this.guest = guest;
    }
  public int getNbr_rooms_reserved() {
        return nbr_rooms_reserved;
    }

    public void setNbr_rooms_reserved(int nbr_rooms_reserved) {
        this.nbr_rooms_reserved = nbr_rooms_reserved;
    }

    public double getTotal_amount() {
        return total_amount;
    }

    public void setTotal_amount(double total_amount) {
        this.total_amount = total_amount;
    }

    public boolean isIsGift() {
        return isGift;
    }

    public void setIsGift(boolean isGift) {
        this.isGift = isGift;
    }

    @Override
    public String toString() {
        return "Booking{" + "id=" + id + ", term=" + term + ", reviewed=" + reviewed + ", bookingDate=" + bookingDate + ", nbr_rooms_reserved=" + nbr_rooms_reserved + ", total_amount=" + total_amount + ", isGift=" + isGift + ", property=" + property + ", guest=" + guest + '}';
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Booking other = (Booking) obj;
        if (this.term != other.term) {
            return false;
        }
        if (this.nbr_rooms_reserved != other.nbr_rooms_reserved) {
            return false;
        }
        if (!Objects.equals(this.bookingDate, other.bookingDate)) {
            return false;
        }
        return true;
    }
}

File: Host_n_Guest/src/BookingBundle/Entity/Booking.php
<?php

namespace BookingBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity
 */
class Booking
{
    /**
     * @ORM\Id
     * @ORM\Column(type="integer")
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    protected $id;

    /**
     * @ORM\Column(type="integer")
     */
    protected $term;

    /**
     * @ORM\Column(type="integer")
     */
    protected $nbReservedRooms;

    /**
     * @ORM\Column(type="integer")
     */
    protected $totalAmount;

    /**
     * @ORM\Column(type="boolean")
     */
    protected $reviewed;

    /**
     * @ORM\Column(type="date")
     */
    protected $bookingDate;

    /**
     * @ORM\ManyToOne(targetEntity="PropertyBundle\Entity\Property")
     */
    protected $property;

    /**
     * @ORM\ManyToOne(targetEntity="UserBundle\Entity\User")
     */
    protected $guest;

    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return mixed
     */
    public function getTerm()
    {
        return $this->term;
    }

    /**
     * @param mixed $term
     */
    public function setTerm($term)
    {
        $this->term = $term;
    }

    /**
     * @return mixed
     */
    public function getNbReservedRooms()
    {
        return $this->nbReservedRooms;
    }

    /**
     * @param mixed $nbReservedRooms
     */
    public function setNbReservedRooms($nbReservedRooms)
    {
        $this->nbReservedRooms = $nbReservedRooms;
    }

    /**
     * @return mixed
     */
    public function getReviewed()
    {
        return $this->reviewed;
    }

    /**
     * @param mixed $reviewed
     */
    public function setReviewed($reviewed)
    {
        $this->reviewed = $reviewed;
    }

    /**
     * @return mixed
     */
    public function getBookingDate()
    {
        return $this->bookingDate;
    }

    /**
     * @param mixed $bookingDate
     */
    public function setBookingDate($bookingDate)
    {
        $this->bookingDate = $bookingDate;
    }

    /**
     * @return mixed
     */
    public function getProperty()
    {
        return $this->property;
    }

    /**
     * @param mixed $property
     */
    public function setProperty($property)
    {
        $this->property = $property;
    }

    /**
     * @return mixed
     */
    public function getGuest()
    {
        return $this->guest;
    }

    /**
     * @param mixed $guest
     */
    public function setGuest($guest)
    {
        $this->guest = $guest;
    }

    /**
     * @return mixed
     */
    public function getTotalAmount()
    {
        return $this->totalAmount;
    }

    /**
     * @param mixed $totalAmount
     */
    public function setTotalAmount($totalAmount)
    {
        $this->totalAmount = $totalAmount;
    }
}
